### ISSUE TITLE
Incorrect safeAreaInsets on tab bar within UITabBarController

### ISSUE THREAD

**Issue by wstyres** [LeoNatan/LNPopupController#370]
#### Description
On notched devices on 12.x and when using a UITabBarController, the tab is initially drawn without safeAreaInsets and appears shrunk. After presenting the popup bar, the tab bar is drawn correctly and even after the popup bar is dismissed. Rotation will also cause the tab bar to be drawn correctly. 

Un-linking the LNPopupController framework from the project will not cause the bug but obviously the popup bar can no longer be called.

Initial launch:
![Simulator Screen Shot - iPhone Xs - 2019-10-27 at 12 54 55](https://user-images.githubusercontent.com/19736790/67638276-fe8f8400-f8b8-11e9-9b4b-535895f3605e.png)

Popup presented:

![Simulator Screen Shot - iPhone Xs - 2019-10-27 at 12 55 27](https://user-images.githubusercontent.com/19736790/67638284-17983500-f8b9-11e9-8798-88842ccff866.png)

No popup after presented:

![Simulator Screen Shot - iPhone Xs - 2019-10-27 at 12 55 37](https://user-images.githubusercontent.com/19736790/67638286-1c5ce900-f8b9-11e9-9a34-317ebd274917.png)

#### Steps to Reproduce
Not able to reproduce in a smaller project unfortunately but the project that it does appear in is available [here](https://github.com/wstyres/Zebra). The LNPopupController is built in a submodule so make sure it is cloned along with it. The tab bar will be shrunk upon first launch.

#### Device, OS and Xcode Versions
Latest Xcode 11.2 beta, latest commit of LNPopupController. I have seen this issue on 12.4 X/Xs/Xr simulators and on physical X/Xs devices on 12.4.

**Comment by wstyres** [comment]
Also happens on Xcode 11.1

**Comment by mamady83** [comment]
in my case this happens only in ios 13. I too would like to know how to solve it

**Comment by LeoNatan** [comment]
Thanks for reporting.

Unfortunately, I will not have time to look at an external project. Would appreciate if you could reproduce in the built-in example project. Perhaps try to present the par in the same place as you present it in your own project.

Thanks

**Comment by wstyres** [comment]
@mamady83 I temporarily resolved it by adding these lines of code to my TabBarController's viewDidAppear to simply force the tab bar to re-layout:
```
//poor hack to get the tab bar to re-layout
if (@available(iOS 11.0, *)) {
    self.additionalSafeAreaInsets = UIEdgeInsetsMake(0, 0, 1, 0);
    self.additionalSafeAreaInsets = UIEdgeInsetsMake(0, 0, 0, 0);
}
```

**Comment by LeoNatan** [comment]
I can't reproduce unfortunately. Taking into account that iOS 14 is around the corner, iOS 13 is already very popular, and you have a workaround, I will leave this issue be and close it.
